intelmq/bots/collectors/mail/lib.py

```diff
@@ -1,50 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
I guess this change is accidentally in this commit/PR as it has nothing to do with issue #792?